packages/contracts/test/UnirepApp.test.ts

```diff
@@ -32,11 +44,15 @@ async function genUserState(id, app) {
 describe('Unirep App', function () {
     let unirep
     let app
+    let hashUserId: string
+    let id: Identity
 
     // epoch length
     const epochLength = 300
-    // generate random user id
-    const id = new Identity()
+    // generate random hash user id
+    before(async () => {
+        ;[hashUserId, id] = createRandomUserIdentity()
+    })
 
     it('deployment', async function () {
         const [deployer] = await ethers.getSigners()
```

Comment:
@iverJisty Can we also put this deploy part into before? Since there is no checks here